less/home-page.less

```
@@ -58,18 +81,33 @@
       }
 
       .btn {
+        .button(10rem, 3.5rem, 2rem);
```

I took out the height and width arguments on that mixin, and changed it to padding. Padding stretches out the clickable surface of your button. The first argument here is our padding. So the top and bottom are 10px, the left and right are 20px. And text-decoration: none gets rid of the underline that's on "a" tags by default to match the design file. Finally, let's nest your hover style inside. 

```suggestion
        .button(10px 20px, 2rem);
        text-decoration: none;
        &:hover {
          color: #23293B;
          background-color: white;
        }
```

---

Padding increasing the clickable area is immensely helpful. Thank you!